bin/ember-template-lint.js

```diff
@@ -333,6 +337,10 @@ async function run() {
     return;
   }
 
+  if (options.updateTodo) {
+    ensureTodoConfig(options.workingDirectory);
+  }
```

Comment:
Do we want to do this speculatively (as opposed to doing it only when we need to read / write)?

Reply:
Say more...

Reply:
I thought it made more sense to only do it during the write, since if ppl don't use todos, why mutate their package.json? I'm not 100% sure that's what you meant...